src/platform/packages/shared/kbn-esql-validation-autocomplete/src/shared/helpers.ts

```diff
@@ -34,6 +34,10 @@ export function isSourceCommand({ label }: { label: string }) {
   return ['FROM', 'ROW', 'SHOW', 'TS'].includes(label);
 }
 
+export function isHeaderCommand({ label }: { label: string }) {
```

Comment:
This function is better to be moved here src/platform/packages/shared/kbn-esql-ast/src/ast/is.ts